models/articles.model.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../db/connection");
+
+exports.fetchArticlesById = (article_id) => {
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1;", [article_id])
+    .then((result) => {
+      if (!result.rows.length) {
+        return Promise.reject({ status: 404, message: "Invalid article ID" });
```

Comment:
In this scenario it wouldn't be an invalid ID, as the ID is valid (i.e. the right datatype), but doesn't exist.